recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}:"
+
+SRC_URI += "file://meson64-kmeta;type=kmeta;destsuffix=meson64-kmeta"
+
+COMPATIBLE_MACHINE_khadas-vim3 = "khadas-vim3"
+
+KERNEL_DEVICETREE_khadas-vim3 = "amlogic/meson-g12b-a311d-khadas-vim3.dtb"
+
+LINUX_VERSION_EXTENSION_append_khadas-vim3 = "-meson64"
```

Comment:
This should be LINUX_VERSION_EXTENSION_append_meson-gx to cover all 64bit socs

Reply:
I agree - but I have only tested with the Vim3.  I didn't want to pollute the namespace until testing could be done.

Happy to change it if you want to though, this is your repo after all :¬)

Reply:
Seeing as this is protected by the COMPATIBLE_MACHINE to only build for the khadas-vim3, I agree with your statement.  Will change accordingly.

Reply:
Yeah, no worry for other boards, it's not the default kernel recipe anyway, I'll handle testing on other boards myself